README.md

```diff
@@ -2,7 +2,7 @@
 
 Implements API for retrieval of symbols and other debug artifacts from symbol store.
 
-Builds are available on the dotnet-tools feed: https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet-tools
+Builds are available on the dotnet-tools feed: https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-tools/NuGet/Microsoft.SymbolStore
```

Comment:
This repo products more than just the Microsoft.SymbolStore package. Microsoft.FileFormats, dotnet-symbol, etc are also produced and published to that feed so it probably should be just: https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-tools

Reply:
maybe you can list the packages in readme then? It just not obvious which package should I reference to use this lib.